jsapp/js/components/transferProjects.module.scss

```diff
@@ -0,0 +1,65 @@
+@use 'scss/sizes';
+@use '~kobo-common/src/styles/colors';
+
+.root {
+  background: colors.$kobo-white;
+  padding: sizes.$x20;
+  margin: sizes.$x24 sizes.$x40;
+  border: 1px solid colors.$kobo-gray-92;
+  border-radius: sizes.$x5;
+}
+
+.bar {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+
+  button {
+    max-width: sizes.$x180;
+    height: sizes.$x38;
```

Comment:
Why overriding `<Button>`'s component height? `l` size is already `38px`.

Reply:
Also please add some classname to the button instead of targeting with `button` selector